### ISSUE TITLE
Figure out distribution of built artifacts

### ISSUE THREAD

**Issue by cybergeek94** [ss31/ss31#15]
ss31 is now configured with [AppVeyor][1]. Additional configuration and logistics research is needed for deploying built artifacts.

[1]: https://ci.appveyor.com/project/cybergeek94/ss31

**Comment by AlmostPete** [comment]
I agree with holding off. Perhaps when we get to the "Oh I can see another person moving around as a programmer art sprite" we can take another look at this.
